.github/workflows/ci_linux.yml

```diff
@@ -0,0 +1,33 @@
+name: CI_Linux
+
+on:
+  pull_request:
```

Comment:
Let's add also a "workflow_dispatch" trigger here, it'll allow us to run builds manually